### PR TITLE
Add Contributing document

### PR DESCRIPTION
**Pull request by egeldenhuys**
I have created a basic Contributing document including the workflow and style guides. Does everyone agree with these standards? Are there more things that should be added to aid collaboration?

See [Starting an Open Source Project](https://opensource.guide/starting-a-project/) for more information.

### CODE REVIEW

**Comment by egeldenhuys** [comment]
A topic for discussion: [gitmoji](https://gitmoji.carloscuesta.me/)

We used gitmojies in some cases. What is the feeling towards gitmoji? Are we going to continue the convention or make it optional?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+1. Separate subject from body with a blank line
+2. Limit the subject line to 72 characters or less
+3. Capitalize the subject line
+4. Do not end the subject line with a period
+5. Use the present tense ("Add feature" not "Added feature") in the subject line
+6. Wrap the body at 72 characters
+7. Use the body to explain what and why vs. how
+
+### Branch Names
+Use the convention `my-new-feature-#62`
```
Comment (FJMentz) [inline CONTRIBUTING.md]: Can we maybe add a convention for if a branch isn't for a specific issue just to clarify all cases?
eg. my-new-feature

Comment (egeldenhuys) [reply CONTRIBUTING.md]: Addressed branch names in new commit :+1:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## Git Workflow
+This project will follow the [Gitflow Workflow].
+
+## Pull Request Process
+1. When your feature/fix is complete create a pull request to merge your barnch
```
Comment (FJMentz) [inline CONTRIBUTING.md]: Barnch -> barn or branch * 😉

Comment (egeldenhuys) [reply CONTRIBUTING.md]: Spell Check may be useful, thank you!

Comment (FJMentz) [reply CONTRIBUTING.md]: XD

**Pull request merged**